tests/health_check.rs

```diff
@@ -0,0 +1,36 @@
+use hilow;
+#[actix_rt::test]
```

Comment:
Please leave a line between the `use` statement and the `#[actix_rt::test]` annotation